### PR TITLE
use protocol-relative URL to load libraries

### PR DESCRIPTION
**Pull request by PeterDaveHello**
http://resume.github.io/ will be redirected to https://resume.github.io/ now, loading libraries via http will make no sense, and the browser will tell users that the page contains some insecure resources,
reference : [The Protocol-relative URL](http://www.paulirish.com/2010/the-protocol-relative-url/)

### CODE REVIEW

**Comment by u2** [comment]
Thank you for your contribution.Apology for long time responding.Could you rebase it, and then I will merge it.

**Comment by PeterDaveHello** [comment]
Sure!

**Comment by PeterDaveHello** [comment]
@u2 updated!

File: index.html [hunk 1]
```diff
@@ -1,14 +1,15 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
+<<<<<<< 78b7ad8f09f1a4349d0b6c8bd6c084801e23fcd4
   <title>GitHub R&eacute;sum&eacute;</title>
```
Comment (u2) [inline index.html]: ?

Comment (PeterDaveHello) [inline index.html]: Sorry, that's a mistake, fixed.

**Comment by PeterDaveHello** [comment]
@u2 what do you think now?

**Comment by u2** [comment]
There are some discussions at #182 #188 .

**Comment by u2** [comment]
Have merged #205 .

**Pull request closed without merging**